recipes/minuit2/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "minuit2" %}
+{% set version = "6.19.01" %}
+{% set hash = "8187338d81a1b3d278b9c5137d6a3893e6455c0255b04ea1b4cb2a3a24b83925" %}
+{% set clang_version = "5.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/root-project/root/archive/v{{ version|replace(".","-") }}.tar.gz
+  sha256: {{ hash }}
```

Comment:
Can you put the hash here instead of using the variable? (As It’s only used once)